include/rocksdb/statistics.h

```diff
@@ -226,6 +226,39 @@ enum Tickers : uint32_t {
   // Number of internal keys skipped by Iterator
   NUMBER_ITER_SKIP,
 
+  // BlobDB specific stats
+  BLOB_DB_NUM_PUT,
+  BLOB_DB_NUM_WRITE,
+  BLOB_DB_NUM_GET,
+  BLOB_DB_NUM_MULTIGET,
+  BLOB_DB_NUM_SEEK,
+  BLOB_DB_NUM_NEXT,
+  BLOB_DB_NUM_PREV,
+  BLOB_DB_NUM_KEYS_WRITTEN,
+  BLOB_DB_NUM_KEYS_READ,
+  BLOB_DB_BYTES_WRITTEN,
+  BLOB_DB_BYTES_READ,
+  BLOB_DB_WRITE_INLINED,
+  BLOB_DB_WRITE_INLINED_TTL,
```

Comment:
Lets add a short description to the new metrics added for Blob, like for BLOB_DB_WRITE_INLINED, BLOB_DB_WRITE_INLINED_TTL, BLOB_DB_GC_NUM_FILES, BLOB_DB_FIFO_NUM_FILES_EVICTED etc.